packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -65,6 +65,14 @@ abstract class ResidentRunner {
       await cleanupAfterSignal();
       exit(0);
     });
+    ProcessSignal.SIGUSR1.watch().listen((ProcessSignal signal) async {
+      printStatus('Caught SIGUSR1');
```

Comment:
The hot reload code that we call immediately prints that it is doing a hot reload.